data/reusables/gpg/copy-ssh-public-key.md

```diff
@@ -23,7 +23,7 @@
 
   {% tip %}
 
-  **Tip:** If `clip` isn't working, you can locate the hidden `.ssh` folder, open the file in your favorite text editor, and copy it to your clipboard.
+  **Tip:** With WSL you should use `clip.exe`. Otherwise if `clip` isn't working, you can locate the hidden `.ssh` folder, open the file in your favorite text editor, and copy it to your clipboard.
```

Comment:
```suggestion
  **Tip:** With Windows Subsystem for Linux (WSL), you can use `clip.exe`. Otherwise if `clip` isn't working, you can locate the hidden `.ssh` folder, open the file in your favorite text editor, and copy it to your clipboard.
```
Just suggesting a small change to write out what WSL stands for, for clarity!